front/pages/api/login.ts

```diff
@@ -20,6 +21,55 @@ async function handler(
 
   switch (req.method) {
     case "GET":
+      let invite = null as null | Workspace;
```

Comment:
```suggestion
      let invite : Workspace | null = null;
```
I had a hard time reading that like so I will suggest this edit, but I understand this is highly subjective. It seems very uncommon though so I would prefer the common form.

Reply:
Will fix :+1: